account-kit/smart-contracts/src/ma-v2/actions/common/types.ts

```diff
@@ -0,0 +1,35 @@
+import type { Address, Hex } from "viem";
+
+export type ModuleEntity = {
+  address: Address;
+  entityId: number;
+};
+
+export type ValidationConfig = {
+  address: Address;
+  entityId: number; // uint8
+  validationFlags: number; // uint32
```

Comment:
This is one way to do it, but when we do so, account devs will have to use a helper function to generate this `validationFlags` byte. IMO it would be slightly better devX for us to accept the inputs to this (the 3 separate flags) and build this byte